app_dart/test/request_handlers/github_webhook_test.dart

```diff
@@ -325,6 +328,160 @@ void main() {
       )).called(1);
     });
 
+    test('Labels Golden changes based on Skia Gold ignore, comments to notify', () async {
+      const int issueNumber = 124;
+      request.headers.set('X-GitHub-Event', 'pull_request');
+      request.body = jsonTemplate('opened', issueNumber, 'master');
+      final Uint8List body = utf8.encode(request.body);
+      final Uint8List key = utf8.encode(keyString);
+      final String hmac = getHmac(body, key);
+      request.headers.set('X-Hub-Signature', 'sha1=$hmac');
+      final RepositorySlug slug = RepositorySlug('flutter', 'flutter');
+
+      when(pullRequestsService.listFiles(slug, issueNumber))
+        .thenAnswer((_) => Stream<PullRequestFile>.value(
+          PullRequestFile()..filename = 'some_change.dart',
+        )
+      );
+
+      when(issuesService.listCommentsByIssue(slug, issueNumber))
+        .thenAnswer((_) => Stream<IssueComment>.value(
+          IssueComment()..body = 'some other comment',
+        ),
+      );
+
+      httpClient.request.response.body = '[{"note" : "124"}]';
```

Comment:
I need to mock this for when the request is made to skia gold for ignores. I've tried a number of different ways, but haven't been able to successfully do it. `FakeHttpClient` isn't a `Mock`, so I'm no sure how to manipulate this single response.
@dnfield, do you have any suggestions?